MainSite/views.py

```diff
@@ -27,12 +29,19 @@ def index(request):
     kandz = var[3]
     eppely = var[4]
 
+    totalPower = float(enphasePower) + float(froniusPower) + float(windPower)
+
     with connection.cursor() as cur:
         cur.execute(sql2)
         var = cur.fetchone()
 
     selPower = var[1]
 
+    with connection.cursor() as cur:
+        cur.execute(sql3)
+        var = cur.fetchone()
```

Comment:
Next week I will be working on the database model. Once I have that created I will switch it over to using the ORM.